cli/submit.py

```diff
@@ -24,23 +28,23 @@ def _get_webin_credentials(self):
         return username, password
 
     def submit_with_webin_auth(self, username, password):
-        print("Proceeding with ENA Webin authentication...")
+        logger.info("Proceeding with ENA Webin authentication...")
 
         headers = {"accept": "*/*", "Content-Type": "application/json"}
         data = {"authRealms": ["ENA"], "username": username, "password": password}
         response = requests.post(self.ENA_AUTH_URL, headers=headers, data=json.dumps(data))
 
         if response.status_code == 200:
-            print("Authentication successful")
+            logger.info("Authentication successful!")
             webin_token = response.text
             response = requests.post(self.SUBMISSION_INITIATE_ENDPOINT,
                                      headers={'Accept': 'application/hal+json',
                                               'Authorization': 'Bearer ' + webin_token})
             response_json = json.loads(response.text)
-            print("Submission ID {} received!!".format(response_json["submissionId"]))
+            logger.info("Submission ID {} received!!".format(response_json["submissionId"]))
             self.upload_submission(response_json["submissionId"], response_json["uploadUrl"])
         else:
-            print("Authentication failed")
+            logger.error("Authentication failed!")
 
     def auth_prompt(self):
         print("Choose an authentication method:")
```

Comment:
I have not changed this to use logger because it is a prompt for user input and it might be a bit odd to display that prompt with the usual logging info like timestamp etc.,